src/main/java/uk/gov/pay/directdebit/payers/dao/GoCardlessCustomerDao.java

```diff
@@ -4,18 +4,12 @@
 import org.jdbi.v3.sqlobject.customizer.Bind;
 import org.jdbi.v3.sqlobject.customizer.BindBean;
 import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
-import org.jdbi.v3.sqlobject.statement.SqlQuery;
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 import uk.gov.pay.directdebit.payers.dao.mapper.GoCardlessCustomerMapper;
 import uk.gov.pay.directdebit.payers.model.GoCardlessCustomer;
 
-import java.util.Optional;
-
 @RegisterRowMapper(GoCardlessCustomerMapper.class)
 public interface GoCardlessCustomerDao {
-    @SqlQuery("SELECT * FROM gocardless_customers g WHERE g.payer_id = :id")
-    Optional<GoCardlessCustomer> findByPayerId(@Bind("id") Long id);
```

Comment:
because we used to create a mandate once we already had a payer / customer, and so we needed findByPayerId to retrieve the gocardless customer. Now we create a mandate at the start of the journey.